src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -324,64 +326,101 @@ private void AddLineBreaks(SourceText text, IList<string> lines)
         }
 
         private bool TryGenerateExteriorTriviaAfterEnter(
-            SyntaxTree syntaxTree,
-            SourceText text,
-            int position,
+            Document newDocument,
+            Document originalDocument,
+            int newPosition,
             int originalPosition,
             ITextBuffer subjectBuffer,
             ITextView textView,
             DocumentOptionSet options,
             CancellationToken cancellationToken)
         {
-            // Find the documentation comment before the new line that was just pressed
-            var token = GetTokenToRight(syntaxTree, originalPosition, cancellationToken);
-            if (!IsDocCommentNewLine(token) || token.SpanStart != originalPosition)
```

Comment:
@rchande I verified that this change alone (slightly modified to correct typos) did fix the issue revealed by your unit test. Here's the code I actually ran in the end:

```csharp
var token = GetTokenToLeft(syntaxTree, position, cancellationToken);
if (!IsDocCommentNewLine(token) || token.Span.End != position)
```